### ISSUE TITLE
Incompatibility with asciidoctor version >= 2.0

### ISSUE THREAD

**Issue by pela9** [riboseinc/asciidoctor-bibliography#99]

**Comment by pela9** [comment]
I get the following error when using asciidoctor-bibliography:

 Unable to activate asciidoctor-bibliography-0.10.0, because asciidoctor-2.0.10 conflicts with asciidoctor (~> 1.5.6) (Gem::ConflictError)

I'm not familiar with ruby/gem, am I doing something wrong or is asciidoctor-bibliography not compatible with the latest asciidoctor?

**Comment by paolobrasolin** [comment]
You did nothing wrong. Thanks for bringing this up @pela9!

Your interpretation is correct: `asciidoctor-bibliography-0.10.0` is not compatible with `asciidoctor-2.*`, due to a strict version requirement.

However I did some tests and it appears that everything is working okay even with `asciidoctor-2.*`, so we can probably just relax the requirement.

I prepared a fix for this in #100; which we'll soon release as `asciidoctor-bibliography-0.10.1`.
